packages/devextreme/js/__internal/grids/grid_core/editing/m_editing_utils.ts

```diff
@@ -31,6 +35,16 @@ export const getEditingTexts = (options) => {
   };
 };
 
+export const generateNewRowTempKey = function (): string {
+  return NEW_ROW_TEMP_KEY_PREFIX + String(new Guid());
+};
+export const isNewRowTempKey = function (key: string): boolean {
+  return !!key
```

Comment:
Isn't that a redundant check? **typeof key === 'string'** quite enough

Reply:
The ```typeof key === 'string'``` pass empty strings
As far as I know, we don't want to pass empty strings here :)

Reply:
Oh, sorry I got your thought just right now.
I agree with @andrewmakarov here

In the other conditions, we check the string length and prefix.
So, the empty string check looks redundant.
Let's remove it :)